src/main/java/lotto/ui/ResultView.java

```diff
@@ -0,0 +1,44 @@
+package lotto.ui;
+
+import lotto.domain.Lottos;
+import lotto.domain.Rank;
+import lotto.domain.Statistics;
+
+import java.text.DecimalFormat;
+
+public class ResultView {
+    private final static DecimalFormat form = new DecimalFormat("#.##");
+
+    private final static String PRINT_STATISTICS_START = "당첨 통계\n---------";
+
+    public static void printPurchaseLotto(Lottos lottos) {
+        System.out.println(lottos.purchaseLotto() + "개를 구매했습니다.");
+        lottos.getLottos()
+                .stream()
+                .forEach(lotto -> System.out.println(lotto.toString()));
```

Comment:
```suggestion
        lottos.getLottos()
                .forEach(lotto -> System.out.println(lotto.toString()));
```
forEach는 stream 연산없이 바로 사용할 수 있습니다. 🙂